config/settings.yml

```diff
@@ -1283,6 +1283,7 @@
         :poll: 10.seconds
       :event_catcher_swift:
         :poll: 10.seconds
+      :event_catcher_openstack_service: "auto"
```

Comment:
```:event_catcher_openstack``` section is for Cloud provider (the name is misleading here, propably for historical reasons).

We need this option to affect all Openstack-like providers (cloud, infra, network, ...) so I'd prefer keep it outside the Cloud provider related section.